package.json

```diff
@@ -10,25 +11,18 @@
   "license": "MIT",
   "author": "Tilde, Inc.",
   "scripts": {
+    "benchmark:start": "pnpm -r --filter @glimmer-workspace/krausest run start",
     "benchmark:build": "node benchmark/bin/build.js",
-    "benchmark:control": "node benchmark/bin/control.js",
-    "benchmark:experiment": "node benchmark/bin/experiment.js",
-    "browserstack:connect": "ember browserstack:connect",
-    "browserstack:disconnect": "ember browserstack:disconnect",
-    "browserstack:results": "ember browserstack:results",
-    "build": "dotenv -- turbo build",
+    "build": "dotenv -c development -- turbo build",
```

Comment:
getting a lot of unhandled external warnings from `pnpm build`.

for, what seems like every package, I get messages like this:
```
@glimmer/syntax:build: unhandled external simple-html-tokenizer { for: '@glimmer/syntax', config: 'undefined' }
@glimmer/syntax:build: unhandled external @glimmer/util { for: '@glimmer/syntax', config: 'undefined' }
@glimmer/syntax:build: unhandled external @glimmer/local-debug-flags { for: '@glimmer/syntax', config: 'undefined' }
@glimmer/syntax:build: unhandled external @glimmer/util { for: '@glimmer/syntax', config: 'undefined' }
@glimmer/syntax:build: unhandled external @glimmer/util { for: '@glimmer/syntax', config: 'undefined' }
@glimmer/syntax:build: unhandled external simple-html-tokenizer { for: '@glimmer/syntax', config: 'undefined'
````

Reply:
on a full cache miss,
```
 Tasks:    19 successful, 19 total
Cached:    0 cached, 19 total
  Time:    1m2.73s 
```

but then... turbo!!!
```
 Tasks:    19 successful, 19 total
Cached:    19 cached, 19 total
  Time:    117ms >>> FULL TURBO
```
WHEEEEEEE

Reply:
This is on a low-power-ish intel i5 11th gen -- 8 core.